app/helpers/storage_resource_helper/textual_summary.rb

```diff
@@ -34,4 +35,14 @@ def textual_logical_free
   def textual_logical_total
     {:label => _("Total Size"), :value => number_to_human_size(@record.logical_total, :precision => 2)}
   end
+
+  def textual_capabilities
+    values = ""
+    JSON.parse(@record.capabilities).each do |capabilities|
```

Comment:
should `@record.capabilities` be checked for nil?

Reply:
It supposed to be {} if there is no capabilities. 
Do you think we need it ?

Reply:
I am not so sure. maybe a test case for this will be good.

Reply:
If there isn't a default on the column or value we need to check for `nil`

Reply:
Also why store JSON here rather than a simple hash?

Reply:
There is a `:default => {}` on the column https://github.com/ManageIQ/manageiq-schema/pull/676/files#diff-d303bb493caec28b84ab3f267197ffd2fa9331922fab8df462780e977920a638R3

Reply:
done